package.json

```diff
@@ -789,30 +824,18 @@
             "default": "latest",
             "description": "Docker image tag to use when starting a local Kafka container"
           },
-          "confluent.localDocker.schemaRegistryImageTag": {
+          "confluent.localDocker.schemaRegistryImageRepo": {
```

Comment:
New setting that didn't exist previously - didn't make much sense to show 3/4 of the settings associated with local Docker images